src/applications/mhv-medications/tests/containers/PrescriptionsPrintOnly.unit.spec.jsx

```diff
@@ -28,6 +43,12 @@ describe('Medications List Print Page', () => {
     sandbox.restore();
   });
 
+  it('renders Correctly even when error is an object', async () => {
+    const screen = setupWithError();
+    const rxName = screen.findByText('Medications | Veterans Affairs');
```

Comment:
Since an error is being rendered, you should assert that some text associated with the error is displayed.

Reply:
Done